src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1146,6 +1146,7 @@ class RPCParser
             {   "submit_multisigned",   &RPCParser::parseSubmitMultiSigned,     1,  1   },
             {   "server_info",          &RPCParser::parseServerInfo,            0,  1   },
             {   "server_state",         &RPCParser::parseServerInfo,            0,  1   },
+            {   "shards",               &RPCParser::parseAsIs,                  0,  0   },
```

Comment:
I tried this CLI command and it said command not supported...shouldn't it be `crawl_shards` to match the RPC handler name?

Reply:
Thanks!